tests/basic-test.sh

```diff
@@ -31,7 +31,7 @@ if is_bare_user_only_repo repo; then
     # In bare-user-only repos we can only represent files with uid/gid 0, no
     # xattrs and canonical permissions, so we need to commit them as such, or
     # we end up with repos that don't pass fsck
-    COMMIT_ARGS="--canonical-permissions"
+    COMMIT_ARGS="--canonical-permissions --no-xattrs"
```

Comment:
I guess this is the bit we should be able to get rid of once we fix the commit side of this.

Reply:
Yes, it is the same workaround we suggested at https://github.com/ostreedev/ostree/issues/2410#issuecomment-896144099.